charts/terraform-cloud-operator/README.md

```diff
@@ -51,18 +51,19 @@ $ helm install demo hashicorp/terraform-cloud-operator \
 
 In the above example, the Operator will watch 3 namespaces in the Kubernetes cluster: `white`, `blue`, and `red`.
 
-If targeting a TFE instance rather than Terraform Cloud, set the API URL using this variable:
+### Install to work with Terraform Enterprise
 
-```
+If targeting a Terraform Enterprise instance rather than Terraform Cloud, set the API endpoint URL using the `operator.tfeAddress` value:
+
+```console
+$ helm install demo hashicorp/terraform-cloud-operator \
+  --version 2.0.0-beta7 \
   --set operator.tfeAddress="https://tfe-api.my-company.com"
 ```
 
-If the TFE instance uses a TLS certificate signed by a non-public authority or "Let's Encrypt", the chain of CAs that can validate 
-that TLS certificate should be installed with the operator by setting the `customCAcertificates` chart value:
+If you encounter a TLS-related issue using the Operator with Terraform Enterprise, it likely could be addressed by using one of the Helm values, `customCAcertificates` or `operator.skipTLSVerify`.
```

Comment:
```suggestion
If you encounter a TLS-related issue using the Operator with Terraform Enterprise, you may need to configure your own CA certificates using the [`customCAcertificates`](https://github.com/hashicorp/terraform-cloud-operator/blob/main/charts/terraform-cloud-operator/values.yaml#L46-L48) value, or by skipping TLS verification using the [`operator.skipTLSVerify`](https://github.com/hashicorp/terraform-cloud-operator/blob/main/charts/terraform-cloud-operator/values.yaml#L31-L32) value..
```

Reply:
I really don't want to add a link to specific lines in the `value.yaml` file due to its mutable nature. What if we just point to the [Value](https://github.com/hashicorp/terraform-cloud-operator/tree/main/charts/terraform-cloud-operator#values) section in the Helm Chart README file?